templates/zerver/realm_creation_link_invalid.html

```diff
@@ -0,0 +1,25 @@
+{% extends "zerver/portico.html" %}
+
+{% block title %}
+<title>{{ _("Organization creation link expired or deactivated") }} | Zulip</title>
+{% endblock %}
+
+{% block portico_content %}
+<div class="error_page">
+    <div class="container">
+        <div class="row-fluid">
+            <img class="hourglass-img" src="/static/images/errors/timeout_hourglass.png" alt=""/>
+            <div class="errorbox">
+                <div class="errorcontent">
+                    <h1 class="lead">{{ _("Whoops. The organization creation link has expired or is not valid.") }}</h1>
+                    <p>
+                        {% trans %}
+                        You can obtain a new link for creating a new organization on <a href="https://zulip.com/new/">Zulip Cloud</a>.
```

Comment:
I'm sceptical about this sentence, but I don't have good ideas for phrasing this better in a way that will serve well both in the zulip.com and self-hosted scenarios

Reply:
Can we say:
```
You can obtain a <a href="/new">new link for creating a new organization</a>.
```